regression/pages/common/utils.py

```diff
@@ -0,0 +1,496 @@
+        return response.content
+    else:
+        msg = "Could not use restful API.  Status code: {0}".format(
+            response.status_code)
+        raise ApiException(msg)
+
+
+def get_enrollment_codes_from_email(coupon_string):
+    """
+    Read enrollment codes and urls from email
+    Ignore first four lines of file as these are just headers
+    Args:
+        coupon_string:
+    Returns:
+        coupon codes and urls:
+    """
+    coupons = {}
+    for i, row in enumerate(coupon_string.splitlines()):
+        # Read the coupons starting from line 5
+        if i > 3:
+            if row:
+                new_row = row.split(',')
+                coupons.update({new_row[0]: new_row[1]})
+    if coupons:
+        return coupons
+    else:
+        return 'Coupons not found'
+
+
+def skip_cleanup_if_test_passed():
+    """
+    Skip cleanup when tests are passing
+    """
+    def decorator(test_function):
+        """
+        Decorator
+        Args:
+            test_function:
+        """
+        @functools.wraps(test_function)
+        def wrapper(self, *args, **kwargs):
+            """
+            Wrapper
+            Args:
+                self:
+                *args:
+                **kwargs:
+            """
+            test_function(self, *args, **kwargs)
+            self.full_cleanup = False
+        return wrapper
+    return decorator
+
+
+def fill_input_fields(page, selectors_and_values_dict):
+    """
+    A helper function to fill multiple fields in a form
+    Args:
+        page:
+        selectors_and_values_dict:
+    """
+    for key, value in selectors_and_values_dict.iteritems():
+        page.q(css=key).fill(value)
+
+
+def select_value_from_drop_down(page, drop_down_name, value):
+    """
+    A helper function to select value from drop down
+    Args:
+        page:
+        drop_down_name:
+        value:
+    """
+    page.wait_for_element_visibility(
+        'select[name={}]'.format(drop_down_name), 'Drop down is visible')
+    page.q(
+        css='select[name={}] option[value="{}"]'.format(drop_down_name, value)
+    ).click()
+    page.wait_for(
+        lambda: page.q(
+            css='select[name={}] option[value="{}"]'.
+            format(drop_down_name, value)
+        ).selected,
+        "Correct value is selected",
+        timeout=DEFAULT_TIMEOUT
+    )
+
+
+def click_checkbox(page, css_selector):
+    """
+    A helper function to click check boxes
+    Args:
+       page:
+       css_selector:
+    """
+    # pylint: disable=unnecessary-lambda
+    page.wait_for_element_visibility(css_selector, 'wait for target checkbox')
+    page_elem = page.q(css=css_selector).results[0]
+    if not page_elem.is_selected():
+        page.browser.execute_script("arguments[0].click();", page_elem)
+        page.wait_for(
+            lambda: page_elem.is_selected(),
+            'Target checkbox is selected'
+        )
+
+
+def remove_spaces_from_list_elements(list_with_spaces):
```

Comment:
@benpatterson 

I am not sure it is a bug or not as it is not impacting any functionality, however if you open the profile page in firefox and chrome and use inspect element on countries list you will see that in firefox Falkland is shown as Falkland Islands [Malvinas] but in chrome it has 2 spaces between island and [Malvinas], it is like Falkland Islands  [Malvinas].

Do you think I should log a bug about it?